homeassistant/helpers/area_registry.py

```diff
@@ -63,15 +62,15 @@ class EventAreaRegistryUpdatedData(TypedDict):
     area_id: str
 
 
-@dataclasses.dataclass(frozen=True, kw_only=True)
+@dataclass(frozen=True, kw_only=True)
 class AreaEntry(NormalizedNameBaseRegistryEntry):
     """Area Registry Entry."""
 
     aliases: set[str]
     floor_id: str | None
     icon: str | None
     id: str
-    labels: set[str] = dataclasses.field(default_factory=set)
+    labels: set[str] = field(default_factory=set)
```

Comment:
This change looks unrelated

Reply:
Just a small stylistic import change, the other registries are already doing it like that, so I wanted to make it more uniform.
If you really want, I can extract it to a separate PR.